examples/step-32/step-32.cc

```diff
@@ -3237,22 +3236,24 @@ namespace Step32
   //
   // The quantities we output here are more for illustration, rather than for
   // actual scientific value. We come back to this briefly in the results
-  // section of this program and explain what one may in fact be interested
-  // in.
+  // section of this program and explain what one may in fact be interested in.
   template <int dim>
   void
   BoussinesqFlowProblem<dim>::Postprocessor::
-  compute_derived_quantities_vector (const std::vector<Vector<double> >              &solution_values,
-                                     const std::vector<std::vector<Tensor<1,dim> > > &solution_gradients,
-                                     const std::vector<std::vector<Tensor<2,dim> > > &/*solution_hessians*/,
-                                     const std::vector<Point<dim> >                  &/*normals*/,
-                                     const std::vector<Point<dim> >                  &/*evaluation_points*/,
-                                     std::vector<Vector<double> >                    &computed_quantities) const
+  compute_derived_quantities_vector
+  (const DataPostprocessorInputs::Vector<dim> &inputs,
+   std::vector<Vector<double> >               &computed_quantities) const
   {
+    const std::vector<Vector<double> > &solution_values = inputs.solution_values;
+    const std::vector<std::vector<Tensor<1,dim> > > &solution_gradients
+      = inputs.solution_gradients;
```

Comment:
same here and probably elsewhere. I'm just not very fond of local variables that are references to other local variables.